### ISSUE TITLE
L4 Core getting very hot

### ISSUE THREAD

**Issue by w0ati** [Dfinitski/N7DDC-ATU-100-mini-and-extended-boards#82]
I had noticed after using the ATU, the case was a little warm. Its sitting on top of the radio so did not give it much thought.
Today after using it I opened it for other reasons. I got the "Allan Bradley" smell right off. So looked further. 
I found the core of L4 was very hot and the paint has burned off (hence the smell). Most of the tuning uses that coil and 
combinations of the air wound coils.
I am using 100W or less.
Is this expected??

**Comment by Dfinitski** [comment]
No. Seems like you have bad cores with large losses.nothing should be burned or has some smell.

**Comment by w0ati** [comment]
I swaped out the core and made a contact with a configuration that used L4 (the new core). It did not appear to get hot. I will monitor it for a while. I am now concerned about all the other cores. The tuner rarely uses any inductor higher than L4. So the other cores in this kit could be bad and I would not know it. I am wondering if I should buy all new cores from a known/dependable supplier.

Sent from Yahoo Mail on Android 
 
  On Thu, Dec 23, 2021 at 5:22 PM, David ***@***.***> wrote:   


No. Seems like you have bad cores with large losses.nothing should be burned or has some smell.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Dfinitski** [comment]
Kitsandparts.com

**Comment by nv1o** [comment]
Isn't it important to wind the cores in the same direction?  I didn't see it mentioned in the instructions, but would think it might be desirable.

**Comment by Dfinitski** [comment]
There is no any assembling instructions. I have added couple photos on the schematic to clear questions like this .

**Comment by nv1o** [comment]
Yes, those were the instructions I was referring to, coil size, number of turns, etc., and yet my question remains unanswered.

**Comment by Dfinitski** [comment]
Ok, there is a photo of tandem match too.

**Comment by soosp** [comment]
Same issue here:
![photo_2022-04-19_15-42-14](https://user-images.githubusercontent.com/24782682/164017452-b8f3287e-dee0-45dd-ad23-ff1d4230b45b.jpg)
![photo_2022-04-19_15-42-19](https://user-images.githubusercontent.com/24782682/164017457-c8f91972-747f-45ea-bb5e-d8f8451d9fcf.jpg)

**Comment by w0ati** [comment]
Fascinating, at some level it's nice to know I am not along (crazy). Seems like the big disadvantage with
the implementation shown in the link is coupling between inductors. Note L2 and L3 are at right angles
to each other and L1 is place off somewhere else. This would be to lower coupling between them. The number of windings and loss (series R) would come in to play at some point.
To be honest, I have stopped relying so much on the ATU and just tune the antenna lengths.

**Comment by Dfinitski** [comment]
That is famous problem connected with bad Chinese "amidon" cores with high losses on RF.
Change them https://kitsandparts.com/store2.php

**Comment by Hobby4life** [comment]
I have measured these cores with the advised amount of windings, but none came nowhere at their given inductance. I have replaced these cores with original Amidon T68-2 cores.. and this looks more promising.
